### PR TITLE
chore: improve generated text for manager docs

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Improve the generated text for the manager docs pages
  - Rewrite two sentences
  - Link to our new discussion form, the old link doesn't lead to the bug report anymore
  - Use simple words and phrases

## Context

Drive by PR.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: tools/docs/manager.ts [hunk 1]
```diff
@@ -131,12 +131,12 @@ sidebar_label: ${displayName}
       md += `Renovate supports updating ${nameWithUrl} dependencies.\n\n`;
       if (defaultConfig.enabled === false) {
         md += '## Enabling\n\n';
-        md += `${displayName} functionality is currently in beta testing so you must opt in to test it out. To enable it, add a configuration like this to either your bot config or your \`renovate.json\`:\n\n`;
+        md += `${displayName} functionality is currently in beta testing, so you must opt-in to test it. To enable it, add a configuration like this to either your bot config or your \`renovate.json\`:\n\n`;
```
Comment (HonkingGoose) [inline tools/docs/manager.ts:134]: Please check that I'm using the correct Semantic Versioning prefix, I think `chore:` is okay, but maybe it should be `fix:`?

Comment (viceice) [reply tools/docs/manager.ts:134]: chore is fine, we don't need a renovate release to update the docs

File: tools/docs/manager.ts [hunk 2]
```diff
@@ -131,12 +131,12 @@ sidebar_label: ${displayName}
...
         md += '```\n';
         md += `{\n  "${manager}": {\n    "enabled": true\n  }\n}`;
         md += '\n```\n\n';
         md +=
-          'If you find any bugs, please [raise a bug report](https://github.com/renovatebot/renovate/issues/new?template=3-Bug_report.md). If you find that it works well, then feedback on that would be welcome too.\n\n';
+          'If you find any bugs, please [create a bug report](https://github.com/renovatebot/renovate/issues/new/choose). If you find that it works well, then let us know too.\n\n';
```
Comment (viceice) [inline tools/docs/manager.ts:139]: We should probably direct to discussions first, as most of the time it's no a bug 😉

Comment (HonkingGoose) [reply tools/docs/manager.ts:139]:
```suggestion
          'If you find any bugs, please [create a new discussion](https://github.com/renovatebot/renovate/discussions/new). If you find that it works well, then let us know too.\n\n';
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**